src/project_file.rs

```diff
@@ -35,6 +35,11 @@ pub struct ObjectMetadata {
 
     /// Notes or comments about the object
     pub notes: Option<String>,
+
+    /// Workaround: Store horizontal justification for OutputString objects
+    /// This is needed because the IOP format may not preserve this correctly
```

Comment:
What do you mean the IOP format may not preserve this correctly? We shouldn't need a workaround. If there's a bug in https://github.com/Open-Agriculture/AgIsoStack-rs/blob/main/src/object_pool/object_attributes.rs#L268. We should fix that instead :)